### PR TITLE
feat/newRoadmap

### PR DESCRIPTION
**Pull request by Fupete**
Editing the /piano-attivita web page to represent what is happening behind the scenes.

### CODE REVIEW

File: _data/menu_items.yml [hunk 1]
```diff
@@ -1,18 +1,16 @@
...
+  - title: Kit
+    url: /kit
+  - title: Modelli
+    url: /modelli
+  - title: Piano attività
+    url: /piano-attivita
   - title: Notizie
     url: /notizie
-  - title: Progetti
-    url: /progetti
```
Comment (bfabio) [inline _data/menu_items.yml:13]: Is this removal on purpose?

Comment (Fupete) [reply _data/menu_items.yml:13]: Would be nice to remove — in the next release — also the 1st item "Home", it is pretty unuseful. What do you think @bfabio ?

Comment (Fupete) [reply _data/menu_items.yml:13]: FYI @danielaiozzo

**Review by bfabio — APPROVED** [review]

**Comment by Fupete** [comment]
💙
